### PR TITLE
[fix] Leave color information in tracked terminal logs

### PR DESCRIPTION
**Pull request by mihran113**
Fix csi(cursor state) handling: do not remove color information as it is handled by UI side.
 Remove from line only cursor up/down/right/left commands.

### CODE REVIEW

File: examples/pytorch_lightning_track.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def cli_main():
     # ------------
     # testing
     # ------------
-    trainer.test(test_dataloaders=test_loader)
```
Comment (alberttorosyan) [inline examples/pytorch_lightning_track.py:106]: is this related to the fix?

Comment (mihran113) [reply examples/pytorch_lightning_track.py:106]: nope, just found out during the testing, and fixed it :)

**Review by alberttorosyan — APPROVED** [review]
lgtm
